RecoJets/JetProducers/src/PileupJetIdAlgo.cc

```diff
@@ -211,9 +211,9 @@ void PileupJetIdAlgo::runMva()
 			internalId_.mva_ = -2.;
 		} else {
 			if(etaBinnedWeights_){
-			  if(std::abs(internalId_.jetEta_)<=2.) internalId_.mva_ = reader_jteta_0_2_->EvaluateMVA( tmvaMethod_.c_str() );
```

Comment:
consider to reimplement this with configurable selection of the eta range (put the readers into a vector and select the right reader based on configurable etaLo,etaHi.

Doesn't this TMVA format support categories? it would be better that the eta-dependence (and whatever else you come up with) gets handled there internally.
This would allow to update more easily.
Right now it is impossible to have two different versions/variants of corrections.